velox/functions/lib/window/tests/WindowTestBase.cpp

```diff
@@ -48,6 +49,80 @@ WindowTestBase::QueryInfo WindowTestBase::buildWindowQuery(
   return {op, functionSql, querySql};
 }
 
+namespace {
+void buildOrderByParameters(
```

Comment:
Nit : This function seems like a substr followed by string split. Rename to splitOrderBy.

Reply:
Updated.